mlir/lib/Dialect/Rock/Tuning/RockTuningImpl.cpp

```diff
@@ -20,6 +20,47 @@
 namespace mlir {
 namespace rock {
 
+// The full space is a brute-force search for attention kernels
+void createAttnTuningRangeBF(TuningParamSet *newSpace, AttentionOp attnOp,
+                             TuningParamSetKind kind) {
+  const std::vector<std::vector<uint32_t>> validRangeAccelGemmParams = {
+      {32, 64, 128, 256},
+      {32, 64, 128, 256},
+      {8, 16, 32, 64},
+      {32, 64, 128, 256},
+      {32, 64, 128, 256},
+      {4, 8, 16},
+      {1}};
```

Comment:
I guess it will result in `forceUnroll ` to be always `1`. Maybe you can use `constexpr int forceUnroll{1}` and remove the 7th loop?

Reply:
done